### ISSUE TITLE
Create chart for powerdns and powerdns-admin

### ISSUE THREAD

**Issue by onedr0p** [billimek/billimek-charts#288]
There's a bunch of charts out there, however there is only one on helm hub:

https://github.com/halkeye-helm-charts/powerdns
https://github.com/halkeye-helm-charts/powerdnsadmin

It would be nice to merge these into one.

**Comment by billimek** [comment]
Partially resolved by #308 ?

**Comment by carpenike** [comment]
@onedr0p -- I think it'd make sense to add the powerdns-admin to this chart and have it toggle on/off with a `.Values.powerdnsadmin.enabled` switch. 

However we end up doing it though, the configuration of PowerDNS-Admin will be manual as the app itself doesn't expose the PowerDNS API configuration per this issue: https://github.com/ngoduykhanh/PowerDNS-Admin/issues/783

**Comment by onedr0p** [comment]
I agree we can have an option to enable and maybe in the readme have some small instructions on how to configure it.
